modules/commerce_bluesnap_currency/src/EventSubscriber/ExchangeRateBluesnap.php

```diff
@@ -46,24 +54,30 @@ public static function sourceId() {
   public function getExternalData($base_currency = NULL) {
     $external_data = [];
     // Prepare for client.
-    $url = self::apiUrl($this->configFactory->get('bluesnap')['mode']);
+    $url = self::apiUrl(
+      $this->config
+        ->get('bluesnap')['mode']
+    );
     $method = 'GET';
     $options['auth'] = [
-      $this->configFactory->get('bluesnap')['username'],
-      $this->configFactory->get('bluesnap')['password'],
+      $this->config
+        ->get('bluesnap')['username'],
+      $this->config
+        ->get('bluesnap')['password'],
```

Comment:
**CHANGE REQUEST**

You need to break to multiple-lines if they exceed the coding standards soft limit i.e. 80 characters. This is not the case here, you can leave `$this->config->get('bluesnap')[*]` in a single line in all 3 cases above.

Reply:
Yep Made the change